update/Propagating_Updates.md

```diff
@@ -0,0 +1,66 @@
+# Propagating Updates
+
+A discussion on updating an existing WSO2 product deployment in a container platform.
+
```

Comment:
This Document depicts important information about updating an existing WSO2 product deployment in a Container Platform.